test/utility/mockApp.ts

```diff
@@ -27,7 +27,7 @@ export interface IApp {
 export const mockAppSpyObj = {
   // Load
   load: jasmine.createSpy("load").and.returnValue(Promise.resolve(null)),
-  validateLoad: jasmine.createSpy("validateLoad").and.callFake(models.validateLoad),
+  validateLoad: jasmine.createSpy("validateLoad").and.callFake(models.validateReportLoad),
```

Comment:
Similar issue here, I'll see how it looks if we add support for dashboard as well.